src/Compilers/CSharp/Test/Semantic/Semantics/GenericConstraintsTests.cs

```diff
@@ -1663,18 +1667,18 @@ public void M()
         public void UnmanagedConstraint_Compilation_ReferenceType()
         {
             CreateCompilation("public class Test<T> where T : class, unmanaged {}").VerifyDiagnostics(
-                // (1,39): error CS8374: The 'unmanaged' constraint cannot be specified with other constraints.
+                // (1,39): error CS8374: The 'unmanaged' constraint must come before any other constraints
```

Comment:
>// (1,39): error CS8374: The 'unmanaged' constraint must come before any other constraints [](start = 16, length = 90)

It would be good to assert that the symbol doesn't have the ```unmanaged``` constraint